src/current/_data/redirects.yml

```diff
@@ -167,7 +167,7 @@
 
 - destination: cockroach-sql.md
   sources: ['use-the-built-in-sql-client.md']
-  versions: ['v19.2', 'v20.1', 'v20.2', 'v21.1', 'v22.2', 'v23.1', 'v23.2']
+  versions: ['v19.2', 'v20.1', 'v20.2', 'v21.1', 'v21.2', 'v21.2', 'v22.2', 'v23.1', 'v23.2']
```

Comment:
```suggestion
  versions: ['v19.2', 'v20.1', 'v20.2', 'v21.1', 'v21.2', 'v22.1', 'v22.2', 'v23.1', 'v23.2']
```

Reply:
As discussed, this is a harmless bug that isn't worth opening a PR to fix -- it will just create the redirect, then clobber it to create it again.